pprof_protobuf/src/protobuf_codec.rs

```diff
@@ -0,0 +1,621 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Why hand writing this? I thought protobuf-build should generate these wrappers for you.

Reply:
`protobuf-build` seems to generate wrappers depending on `rust-protobuf`. I don't want to bring in `rust-protobuf` dependency for prost user.

Reply:
Then why not adapt prost syntax instead? Most of fields generated by protobuf are public, they can be access directly, no need to implement get/set/clear method.

Reply:
The only necessary wrapper should be ser/deser.

Reply:
But also `::protobuf::RepeatedField`. Actually this file is not hand written. It was nearly the same with `rust-protobuf`'s generating file. And then I left only getter and setter and replace `::protobuf::RepeatedField` related part with `Vec` and macro call.

Reply:
The point is how to make it maintainable. Is it still an easy job for others to update the proto?